index.html

```diff
@@ -134,6 +134,60 @@ <h5>SEE THE WHOLE PROGRAM</h5>
 				<a class="join-button" href=""> Join LOTR Celebration 2022</a>
 			</section>
 		</main>
+		<section id="featured-actors" class="featured-actors">
+			<h2>Featured Characters</h2>
+			<hr />
+		</section>
+		<div class="see-more">
+			<button id="see-more-button" type="button">MORE</button>
+		</div>
+		<footer>
+			<section class="partners">
+				<div class="mobile-container">
+					<h3>Partner</h3>
+					<hr />
+					<div class="footer-logos">
```

Comment:
- [ ] Please make sure that the cards displayed in the about page appear side by side on desktop view to match the design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/169329913-4c2509c6-e15a-48e3-8003-9928d3250d74.png) | ![image](https://user-images.githubusercontent.com/62290545/169329666-c4fd13ba-bc25-4c68-b484-14fa4ae61ffc.png) |